core/bin/dust_api.rs

```diff
@@ -10,6 +10,23 @@ use axum::{
     routing::{delete, get, patch, post},
     Router,
 };
+use futures::future::try_join_all;
```

Comment:
started organizing the imports in a consistent way on files I touch, but we might want to start linting this (pretty messy currently)

Reply:
Maximal bundling I presume? If you can find a linter for it let's do it.

Ideally we lint also the fact that we go back to `crate` in the code base. 

If you find linters for one or the other feel free to roll out ofc

Reply:
I went with:
- non-crate imports at top, then empty line then crate imports
- all crate imports grouped together
- no super (we tend to do that already, but we still have some, probably due to vscode auto-importing as super). 

But I don't have a strong opinion about any of these, I'm happy with whatever (but I dislike the lack of consistency)

I'll look for a linter